provision/core/php/provision.sh

```diff
@@ -53,6 +53,7 @@ function php_register_packages() {
 
   # XDebug
   VVV_PACKAGE_LIST+=(
+    "php${VVV_BASE_PHPVERSION}-xdebug"
```

Comment:
hmmm this shouldn't be needed what with the line below, but I don't see the harm, I'll test this shortly

Reply:
Without that line, clone VVV, run `vagrant up`, ssh to it and run `xdebug_on`. It will give the error below. The `php-xdebug` package doesn't seem to actually function as a metapackage or version-agnostic installation as you might expect.

![Screenshot-20210302-13:41:17](https://user-images.githubusercontent.com/36433/109717796-86079a00-7b74-11eb-8e02-6a405fbbf27b.png)